deployment/docker/Dockerfile

```diff
@@ -1,35 +1,57 @@
-FROM ubuntu:24.04 AS builder
```

Comment:
Because there is no need - previously the multistage build was so we could build pdf2htmlex in ubuntu (the only place at the time where we could successfully build it) and then copy it into debian-slim. Now that we can successfully build it in debian-slim, there's no need for a multistage build

Reply:
The goal here is not to decrease the image size, the goal is to build the pdf2htmlEX binary in the same environment where it will be used to prevent segfault runtime errors. Previously, the only reason we used multistage build was that we couldn't actually build pdf2htmlEX in a non-Ubuntu environment (not to reduce image size); now that we can build it from source, we should just use one base image.

Reply:
hmm OK that's fine, I'll make this change